Cargo.toml

```diff
@@ -18,39 +18,51 @@ inherits = "release"
 
 [dependencies]
 candid = { workspace = true }
+ethnum = { workspace = true }
 evm_rpc_types = { path = "evm_rpc_types" }
+futures = { workspace = true }
 getrandom = { workspace = true }
 ic-canisters-http-types = { workspace = true }
+ic-crypto-sha3 = { git = "https://github.com/dfinity/ic", rev = "release-2024-09-12_01-30-base" }
+ic-ethereum-types = { git = "https://github.com/dfinity/ic", rev = "release-2024-09-12_01-30-base" }
```

Comment:
The plan is to get rid of these links and get the dependencies from crates.io in the near future, right?

Reply:
yes (see last paragraph in PR description)